t/no_pod.t

```diff
@@ -0,0 +1,39 @@
+use strict;
+use warnings;
+
+use Dir::Self;
+use lib __DIR__ . '/..';
+use Test::More;
+use Test::Builder::Tester;
+use Module::Path 'module_path';
+
+use Test::Pod::CoverageChange;
+use t::Nopod;
+
+# Initializing variables
+my $test_module = "t::Nopod";
+my $test_module_path = 't/Nopod.pm';
+my $main_module_path = module_path('Test::Pod::CoverageChange');
+
+subtest 'Module with no pod, unexpected' => sub {
+    test_out("not ok 1 - Pod coverage on $test_module");
+    test_out("not ok 2 # TODO There is no POD in the file $test_module_path.");
+    test_out("#   Failed (TODO) test at $main_module_path line 175.");
```

Comment:
@zakame can you help me with this please? I want to get rid of this filename and line number but I've couldn't handle it. Do you have any idea to do that?

Reply:
So if I read this right, this TODO fail is raised via https://github.com/binary-com/perl-Test-Pod-CoverageChange/blob/9026e1d4ea3aa1cde400c73b699e3e7bbd1757c8/lib/Test/Pod/CoverageChange.pm#L175-L180 and what you're trying to do is to raise this TODO fail from the context of this test script, not from this `CoverageChange` module, a la Carp.  Per https://metacpan.org/pod/Test::More#Extending-and-Embedding-Test::More, you probably might need to check via the underlying `Test::Builder` instead and use https://metacpan.org/pod/Test::Builder#todo_start and https://metacpan.org/pod/Test::Builder#todo_skip ...

Reply:
@zakame I change these parts. Thanks for your great comment.